designer/client/src/list/ListItemEdit.test.tsx

```diff
@@ -148,4 +148,22 @@ describe('ListItemEdit', () => {
       'my condition'
     )
   })
+
+  test('message is displayed if the list item is referenced', () => {
+    render(
+      <RenderListEditorWithContext
+        data={data}
+        initialName="myList"
+        initialItemText="text a"
+      >
+        <ListItemEdit />
+      </RenderListEditorWithContext>
+    )
+
+    const $itemDeleteMessage = screen.getByText(
+      'This list item cannot be deleted as it is referenced in: "my condition"'
```

Comment:
Hey Dave, great work. Can we clarify what `"my condition"` is?

Given another name, it's not clear what references the list item (could be a component?)

```suggestion
      'This list item cannot be deleted as it is referenced by condition: "my condition"'
```

Reply:
Resolved with b06df58be0ff0f812620f34e625283411e7d2106